Octokit.Reactive/Clients/IObservablePullRequestReviewClient.cs

```diff
@@ -0,0 +1,190 @@
+        IObservable<PullRequestReview> GetAll(string owner, string name, int pullRequestId);
+
+        /// <summary>
+        /// Gets reviews for a specified pull request.
+        /// </summary>
+        /// <remarks>https://developer.github.com/v3/pulls/reviews/#list-reviews-on-a-pull-request</remarks>
+        /// <param name="repositoryId">The Id of the repository</param>
+        /// <param name="pullRequestId">The pull request number</param>
+        IObservable<PullRequestReview> GetAll(long repositoryId, int pullRequestId);
+
+        /// <summary>
+        /// Gets reviews for a specified pull request.
+        /// </summary>
+        /// <remarks>https://developer.github.com/v3/pulls/reviews/#list-reviews-on-a-pull-request</remarks>
+        /// <param name="owner">The owner of the repository</param>
+        /// <param name="name">The name of the repository</param>
+        /// <param name="pullRequestId">The pull request number</param>
+        /// <param name="options">Options for changing the API response</param>
+        IObservable<PullRequestReview> GetAll(string owner, string name, int pullRequestId, ApiOptions options);
+
+        /// <summary>
+        /// Gets reviews for a specified pull request.
+        /// </summary>
+        /// <remarks>https://developer.github.com/v3/pulls/reviews/#list-reviews-on-a-pull-request</remarks>
+        /// <param name="repositoryId">The Id of the repository</param>
+        /// <param name="pullRequestId">The pull request number</param>
+        /// <param name="options">Options for changing the API response</param>
+        IObservable<PullRequestReview> GetAll(long repositoryId, int pullRequestId, ApiOptions options);
+
+        /// <summary>
+        /// Gets a single pull request review by ID.
+        /// </summary>
+        /// <remarks>https://developer.github.com/v3/pulls/reviews/#get-a-single-review</remarks>
+        /// <param name="owner">The owner of the repository</param>
+        /// <param name="name">The name of the repository</param>
+        /// <param name="pullRequestId">The pull request review comment number</param>
+        /// <param name="reviewId">The pull request review number</param>
+        IObservable<PullRequestReview> GetReview(string owner, string name, int pullRequestId, int reviewId);
+
+        /// <summary>
+        /// Gets a single pull request review by ID.
+        /// </summary>
+        /// <remarks>https://developer.github.com/v3/pulls/reviews/#get-a-single-review</remarks>
+        /// <param name="repositoryId">The Id of the repository</param>
+        /// <param name="pullRequestId">The pull request review comment number</param>
+        /// <param name="reviewId">The pull request review number</param>
+        IObservable<PullRequestReview> GetReview(long repositoryId, int pullRequestId, int reviewId);
+
+        /// <summary>
+        /// Creates a comment on a pull review.
```

Comment:
Is this right?  This method actually creates the pull request review (not just a comment), right?